tiledb/ml/models/pytorch.py

```diff
@@ -19,16 +19,10 @@ class PyTorchTileDB(TileDBModel):
     TileDB arrays and load PyTorch models from TileDB arrays.
     """
 
-    def __init__(self, model: Optional[Module] = None, *args, **kwargs):
-        super(self.__class__, self).__init__(*args, **kwargs)
-        self.model = model
-
-        if self.namespace:
-            self.file_properties = self.set_file_properties(
-                framework="PYTORCH",
-                framework_version=torch.__version__,
-                preview=self.preview(),
-            )
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            framework="PYTORCH", framework_version=torch.__version__, *args, **kwargs
```

Comment:
Not sure this works if called with positional *args following keyword args. It would also break if the user passes framework and/or framework_version.

It would be safer and clearer (for documentation purposes) to change the signature from `(*args, **kwargs)` to the actual signature. Yes there is some repetition with the base class signature but it's not an exact copy (e.g. `model` is not a Union here) so it's worth spelling it out.

Reply:
Thought of another alternative: make framework & framework_version class attributes since they don't vary by instance. In this case you don't need to override `__init__`:

```py
class PyTorchTileDB(TileDBModel):
    Framework = "PYTORCH"
    FrameworkVersion = torch.__version__
```

Reply:
Will try the 2nd ;)

Reply:
Works nice :)